capstone/src/main/webapp/script.js

```diff
@@ -92,16 +92,16 @@ function zipControl(controlDiv, map) {
 
   //* *button functionality */
   controlUI.addEventListener('click', function() {
-    zipClicks++;
-    if (zipClicks%2==1) {
+    zipClicked = !zipClicked;
```

Comment:
May have to change some variable names to make it more clear. For example this variable name can be showZipCodes